extensions/blocks/opentable/opentable.php

```diff
@@ -72,6 +73,19 @@ function load_assets( $attributes ) {
 		$classes
 	);
 	$content = '<div class="' . esc_attr( $classes ) . '">';
+
+	// OpenTable's wide style has fixed widths within the embedded iframe.
+	// To improve the mobile experience we'll add a standard style widget as
+	// well and use CSS to switch display between the wide style and it.
+	if ( 'wide' === $style ) {
+		$fallback_attributes = array_merge( $attributes, array( 'style' => 'standard' ) );
+		$fallback_url        = build_embed_url( $fallback_attributes );
+		$content            .= '<div class="mobile-fallback">';
+		// phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript
+		$content .= '<script type="text/javascript" src="' . esc_url( $fallback_url ) . '"></script>';
```

Comment:
Do we need to load both this one and the default one all the time?

I wonder if we should / could only load one of them?

Reply:
If we ignore the possibility of users resizing their browser window, we could potentially only load one. Using JS to dynamically add or modify a fallback widget was a little clunky as well and seemed to overcomplicate things.

Do you think ignoring resized windows and just checking for a mobile user agent using `jetpack_is_mobile` would be sufficient?

Reply:
> Do you think ignoring resized windows and just checking for a mobile user agent using jetpack_is_mobile would be sufficient?

That's not something we do in blocks today, but we do it in other places in Jetpack. I'd personally say that's enough for most scenarios.